lib/records/ccda.rb

```diff
@@ -58,7 +58,8 @@ def self.basic_info(entity, ccda_record)
         # patient.effective_time
         patient.race = { 'name' => entity[:race].to_s.capitalize, 'code' => RACE_ETHNICITY_CODES[entity[:race]] }
         patient.ethnicity = { 'name' => entity[:ethnicity].to_s.capitalize, 'code' => RACE_ETHNICITY_CODES[entity[:ethnicity]] }
-        unless entity.alive?
+
+        if !entity.alive? && entity.record_synthea.patient_info[:deathdate] <= end_time
```

Comment:
FYI, you should be able to just write `if !entity.alive?(end_time)`, but this is also fine.